docs/developers/build-starrocks/Build_in_docker.md

```diff
@@ -2,15 +2,15 @@
 
 ## Image
 
-The dev environment image was named as `dev-env`, and was a CentOS7 based image, since branch-2.5, Ubuntu-22.04 is recommended over CentOS7 distribution, the image is renamed to `dev-env-ubuntu` accordingly.
+The name of the development environment images is of the form `dev-env-xxx`. If you use Ubuntu 22.04, you can download the image initiated with `dev-env-ubuntu`, or if you use CentOS 7, you can download the image initiated with `dev-env-centos`.
```

Comment:
The name of the development environment images is in the form of `dev-env-xxx`. If you use Ubuntu 22.04, you can download the image initiated with `dev-env-ubuntu`. If you use CentOS 7, you can download the image initiated with `dev-env-centos`.